### PR TITLE
Added a precondition to format SD card

### PR DESCRIPTION
**Pull request by alohit**
Precondition to format SD card added

### CODE REVIEW

File: docs/platform/developing/flashing-rpi.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ IoT Setup Manager supports the following operating systems:
 -   Windows 10
 -   Ubuntu 18.04 LTS and later
 
+## SD card set up
+
+The SD card must be formatted before flashing image into it.
```
Comment (surbhig12) [inline docs/platform/developing/flashing-rpi.md:27]:
```suggestion
The SD card must be formatted before flashing the image into it.
```

**Review by surbhig12 — COMMENTED** [review]
Minor comment, please verify.

**Comment by surbhig12** [comment]
@alohit, a gentle reminder. Please update the PR.

File: docs/platform/developing/flashing-rpi.md (hunk 1 above)
Comment (alohit) [reply docs/platform/developing/flashing-rpi.md]: Thanks for you review. This comment is fixed.

**Review by surbhig12 — APPROVED** [review]
Thank you for incorporating the comments.

File: docs/platform/developing/flashing-rpi.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ IoT Setup Manager supports the following operating systems:
 -   Windows 10
 -   Ubuntu 18.04 LTS and later
 
+## SD card set up
```
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/platform/developing/flashing-rpi.md:25]: @mijin85cho, I agree.

@alohit, please incorporate this comment.

**Review by surbhig12 — COMMENTED** [review]
Minor comment. Please update.

File: docs/platform/developing/flashing-rpi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alohit) [reply docs/platform/developing/flashing-rpi.md]: Updated. Thanks for review.

**Review by surbhig12 — APPROVED** [review]
Thank you for the update.

**Comment by annie-samsung** [comment]
@mijin85cho, this PR seems to have conflicts could you merge this one or let me know how to go about?

**Comment by mijin85cho** [comment]
> @mijin85cho, this PR seems to have conflicts could you merge this one or let me know how to go about?

Hello, @annie-samsung 

I think you mean the "This branch cannot be rebased due to conflicts".
If even there is no conflicted page list but merging is blocked, in that case please try merging with "Create a merge commit".
You can select that option by clicking the drop down menu next to the "Rebase and merge".

**Comment by annie-samsung** [comment]
> > @mijin85cho, this PR seems to have conflicts could you merge this one or let me know how to go about?
> 
> Hello, @annie-samsung
> 
> I think you mean the "This branch cannot be rebased due to conflicts".
> If even there is no conflicted page list but merging is blocked, in that case please try merging with "Create a merge commit".
> You can select that option by clicking the drop down menu next to the "Rebase and merge".

@mijin85cho 
Thank you, will do.

**Pull request merged**